query-service-impl/src/test/java/org/hypertrace/core/query/service/pinot/QueryRequestToPinotSQLConverterTest.java

```diff
@@ -1120,14 +1242,19 @@ private QueryRequest buildSimpleQueryWithFilter(Filter filter) {
     return builder.build();
   }
 
-  private void assertPQLQuery(QueryRequest queryRequest, String expectedQuery) {
+  private void assertPQLQuery(
+      QueryRequest queryRequest, String expectedQuery, ViewDefinition viewDefinition) {
```

Comment:
if we're going to require passing this in (which is fine), we shouldn't make it a global static.

Reply:
This was because we need service view definition for testing avgrate changes. Rest all use this same view definition.

Reply:
resolved